demisto_sdk/commands/validate/validators/PA_validators/PA131_is_default_datasource.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Pack, Integration]
+
+
+class IsDefaultDataSourceProvidedValidator(BaseValidator[ContentTypes]):
+    error_code = "PA131"
+    description = "Validate that the pack_metadata contains a default datasource if there are more than one datasource."
+    rationale = "Wizards and other tools rely on the default datasource to be set."
+    error_message = "Pack metadata does not contain a default datasource. Please fill in a default datasource."
+    related_field = "defaultDatasource"
+    is_auto_fixable = False
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        datasource_count = 0
+        for content_item in content_items:
+            if content_item == Integration:
+                fetch_incidents_command = False
+                for command in content_item.commands:  # type: ignore
+                    if command.name == "fetch-incidents":
```

Comment:
please change this check to the check if "isfetch: true" "isFetchSamples: true" exists in the yml.
note that in isFetchSamples: true we need to filter out feeds.